recipes/dfam/meta.yaml

```diff
@@ -0,0 +1,21 @@
+about:
+  home: dfam.org
+  license: 'GPL'
+  summary: 'The Dfam database is a collection of Repetitive DNA element sequence alignments, hidden Markov models (HMMs) and matches lists for complete Eukaryote genomes'
+
+build:
+  number: 0
+
+package:
+  name: dfam
+  version: '2.0'
+
+requirements:
+  run:
+      - perl-threaded
+      - hmmer
+      - repeatmasker
+
```

Comment:
You should include test for dfamscan.pl and the download-script. At least if these scripts are executable.